méli-mélo/th-empathy/style.css

```
@@ -0,0 +1,83 @@
+/* Thematic purple */
+.btn-empathy.btn-primary,
+.empathy.wb-steps.quiz .btn-primary {
+    background-color: #5C2790;
+}
+
+/* Purple button hover and focus state */
+.btn-empathy.btn-primary:hover,
+.btn-empathy.btn-primary:focus,
+.btn-empathy.btn-primary:active,
+.empathy.wb-steps.quiz .btn-primary:hover,
+.empathy.wb-steps.quiz .btn-primary:focus,
+.empathy.wb-steps.quiz .btn-primary:active {
+    background-color: #D6B9EC;
+    color: #5c2790;
+}
+
+/* Light Purple button default state */
+.btn-empathy.btn-default,
+.empathy.wb-steps.quiz .btn-default {
+    background-color: #FFF;
+    border-color: #5C2790;
+    color: #5C2790;
+}
+/* Light Purple button hover and focus state */
+.btn-empathy.btn-default:hover,
+.btn-empathy.btn-default:focus,
+.btn-empathy.btn-default:active,
+.empathy.wb-steps.quiz .btn-default:hover,
+.empathy.wb-steps.quiz .btn-default:focus,
+.empathy.wb-steps.quiz .btn-default:active  {
+    background-color: #D6B9EC;
+    border-color: #471F70;
+    color: #471F70;
+}
+
+/* Thematic Heading 1 */
+h1#wb-cont.empathy {
+    border-bottom: 0.18em solid #5C2790;
+    border-image: linear-gradient(to right, #5C2790 71px, transparent 71px);
+    border-image-slice: 1;
+}
+
+/*  Thematic Steps Quiz */
+
+/* Thematic progress bar customization */
+.empathy.wb-steps.quiz progress {
+    /* Reset the default browser styles */
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    border: none;
```

The "completed" color has a poor contrast with the white surrounding background of 1.45:1. I propose to include a little purple border like:

```suggestion
    border: #5c2790 solid 1px;
```

Or to use a color with a contrast greater or equal to 3:1 as per the non-text contrast SC requirement